### PR TITLE
feat: adds test account operation

### PR DESCRIPTION
**Pull request by sravya-yelleti**
Please review test accounts operations

### CODE REVIEW

**Comment by sravya-yelleti** [comment]
@moltar please check the PR

**Comment by moltar** [comment]
@sravya-yelleti looks like some tests are failing

**Comment by moltar** [comment]
@nguyentoanit please see if our eslint setup is broken? Errors seem to be relating to that.

**Comment by nguyentoanit** [comment]
> @nguyentoanit please see if our eslint setup is broken? Errors seem to be relating to that.

Ok. Let me check. 👀

**Comment by moltar** [comment]
@nguyentoanit any updates?

**Comment by nguyentoanit** [comment]
@sravya-yelleti : Please rebase and resolve conflicts.
I have added a few updates for eslint to solve failing tests.
After that, we will review your PR soon.

Thanks.

**Comment by sravya-yelleti** [comment]
@moltar please check PR

**Review by nguyentoanit — APPROVED** [review]

**Comment by nguyentoanit** [comment]
> This branch cannot be rebased due to conflicts

@sravya-yelleti : Please pull and rebase latest code in `master` branch.

**Comment by sravya-yelleti** [comment]
@nguyentoanit it says up to date 
Screenshot 2022-07-20 141022

**Comment by nguyentoanit** [comment]
> @nguyentoanit it says up to date Screenshot 2022-07-20 141022

@sravya-yelleti : Have you tried to pull and rebase from original repo? It looks like you were pulling from forked repo? 🤔

File: test/operations/test-accounts/test-accounts-operation.test.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import { OperationProvider } from '../../../src/operations/operation-provider'
+import { AccountOperation } from '../../../src/operations/test-accounts/test-account-operation'
+import { httpClientFactory } from '../../http-client-factory'
+import { Account } from '../../../src/operations/test-accounts/types'
+import { AmazonMarketplaceAdvertisingCountryCode } from '@scaleleap/amazon-marketplaces'
+import { delay } from '../../test-utils'
+
+jest.setTimeout(15000)
+
+describe('AccountOperation', () => {
```
[the remark this answers is not in the document]
Comment (moltar) [reply test/operations/test-accounts/test-accounts-operation.test.ts]: Skip for now, log an issue to fix it

[the remark this answers is not in the document]
Comment (moltar) [reply test/operations/test-accounts/test-accounts-operation.test.ts]: @sravya-yelleti Please add `.skip` for now as we are not setup for this test now.

**Pull request merged**